h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2131,6 +2110,14 @@ protected Submodel computeSubmodel(int i,double lambda, double nullDevTrain, dou
     
     @Override
     public void computeImpl() {
+      try {
+        doCompute();
+      } finally {
+        _model.unlock(_job);
```

Comment:
Can _model be null at this point?

Reply:
no, it is assigned to in init() and never set to anything else

Reply:
you caught me there, and so did the smoke tests, apperently yes, the `_model` can be null here, fixed

Reply:
_model is instantiated right after init().  If something goes wrong in init(), _model can be null.